include/eblob/blob.h

```diff
@@ -793,6 +798,7 @@ int eblob_defrag(struct eblob_backend *b);
 int eblob_periodic(struct eblob_backend *b);
 int eblob_inspect(struct eblob_backend *b);
 
+int eblob_defrag_init_stats(struct eblob_backend *b);
```

Comment:
I'd made it static, it's just a conviniece function, not a part of API

Reply:
Ah, I see we use in tests, ok

Reply:
Yeah that's the only reason.